changelog/10589.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Allow users to override repr maxsize value (240) by defining
+``PYTEST_DEFAULT_REPR_MAX_SIZE`` environment variable.
```

Comment:
We should document this in the reference together with other environment variables:

https://github.com/pytest-dev/pytest/blob/f6adebb99064ad2a894ea7c04a1257aba5876e21/doc/en/reference/reference.rst?plain=1#L1050-L1053

It would be nice to include in the description an example of a use case (I'm guessing this is interesting to configure while constructing/testing external libraries in a CI environment?).